libs/common-services/src/main/java/uk/gov/di/ipv/core/library/journeys/JourneyUris.java

```diff
@@ -42,7 +42,7 @@ private JourneyUris() {
     public static final String JOURNEY_NOT_FOUND_PATH = "/journey/not-found";
     public static final String JOURNEY_OPERATIONAL_PROFILE_REUSE_PATH =
             "/journey/operational-profile-reuse";
-    public static final String JOURNEY_PENDING_PATH = "/journey/pending";
+    public static final String JOURNEY_F2F_PENDING_PATH = "/journey/pending";
```

Comment:
Something to consider: what to do when they return from with a pending dcmaw async cri response.